cms/static/js/views/course_outline.js

```diff
@@ -220,9 +220,11 @@ define(['jquery', 'underscore', 'js/views/xblock_outline', 'common/js/components
                     event.preventDefault();
                     this.publishXBlock();
                 }.bind(this));
-                element.find('.highlights-button').click(function(event) {
-                    event.preventDefault();
-                    this.highlightsXBlock();
+                element.find('.highlights-button').on('click keydown', function(event) {
+                    if (event.type === 'click' || event.keyCode === 13) {
```

Comment:
Do you need to check for both keyCode 10 and 13?  I see some of our code checking for both in some cases when looking for the Enter key.

Reply:
Oh, the space key should activate a button as well, but the keyCode for space is 32. Is that what you were referring to?